lib/srv/alpnproxy/conn_test.go

```diff
@@ -154,20 +156,18 @@ func TestPingConnection(t *testing.T) {
 			}()
 		}
 
-	readLoop:
 		for i := 0; i < nWrites; i++ {
 			var aggregator []byte
-			for {
+			for j := 0; j < 3; j++ {
```

Comment:
nit: Can you add a comment why 3 or even better, create a constant with some meaningful name?